File: backend/src/main/java/unam/ciencias/ids/playbit/payload/request/EnrollDeleteRequest.java
package unam.ciencias.ids.playbit.payload.request;

import jakarta.validation.constraints.NotBlank;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import unam.ciencias.ids.playbit.models.Tournament;
import unam.ciencias.ids.playbit.models.User;


@NoArgsConstructor
@AllArgsConstructor
@Getter @Setter
public class EnrollDeleteRequest {
    @NotBlank
    private User user;

    @NotBlank
    private Tournament tournament;

}

File: android/Playbit_new/app/src/main/java/mx/ids/playbit/model/user/EnrollDeleteRequest.kt
package mx.ids.playbit.model.user

import mx.ids.playbit.model.User
import mx.ids.playbit.model.tournament.Tournament

/**
 * Data class for EnrollDeleteRequest
 * @author Leonardo Aguilar Rodríguez
 * */
data class EnrollDeleteRequest(
    val user: User,
    val tournament: Tournament
)